integreat_cms/cms/models/users/user.py

```diff
@@ -103,6 +104,22 @@ class User(AbstractUser, AbstractBaseModel):
             "Will be set to true once the user dismissed the page tree tutorial"
         ),
     )
+    totp_key = models.CharField(
+        default=None,
+        null=True,
+        blank=True,
+        max_length=128,
+        verbose_name=_("TOTP key"),
+        help_text=_("Will be used to generate TOTP keys"),
+        validators=[MinLengthValidator(64)],
```

Comment:
Hmm well ok, I guess my idea with the validator didn't make sense since we're never actually validating this value.
We're generating keys of length 32 now, so this would be always invalid, but since we're not calling `full_clean()`, it does not check anything... I guess then we can also remove the check since it creates a deceptive impression of security.
```suggestion
```